#! /bin/sh -e
## 004_configure.in.dpatch by <unknown>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Remove "debian/Makefile" from the original configure.on file

if [ $# -lt 1 ]; then
    echo "`basename $0`: script expects -patch|-unpatch as argument" >&2
    exit 1
fi

[ -f debian/patches/00patch-opts ] && . debian/patches/00patch-opts
patch_opts="${patch_opts:--f --no-backup-if-mismatch} ${2:+-d $2}"

case "$1" in
    -patch) patch -p1 ${patch_opts} < $0;;
    -unpatch) patch -R -p1 ${patch_opts} < $0;;
    *)
        echo "`basename $0`: script expects -patch|-unpatch as argument" >&2
        exit 1;;
esac

exit 0

@DPATCH@
diff -urNad shadow-4.0.3/configure.in /tmp/dpep.ZnE7gx/shadow-4.0.3/configure.in
--- shadow-4.0.3/configure.in	2005-05-11 09:56:26.668669556 +0200
+++ /tmp/dpep.ZnE7gx/shadow-4.0.3/configure.in	2005-05-11 09:56:47.679241252 +0200
@@ -306,7 +307,6 @@
 	lib/Makefile
 	src/Makefile
 	contrib/Makefile
-	debian/Makefile
 	etc/Makefile
 	etc/pam.d/Makefile
 	shadow.spec])
